static/app/components/performance/spanSearchQueryBuilder.tsx

```diff
@@ -94,7 +94,7 @@ export function SpanSearchQueryBuilder({
       {
         value: 'custom_fields',
         label: 'Custom Tags',
-        children: Object.keys(customTags),
+        children: Object.keys(customTags ?? {}),
```

Comment:
Due to the way `useSpanFieldSupportedTags` and `useSpanFieldCustomTags` are built, this variable (`customTags`) will actually never be nullish. The reason why typescript thinks `customTags` may be nullish is because the return is typed to `UseApiQueryResult<TagCollection, RequestError>`, which is not the actual true type.

I think if fix the type issue in the hook (see comments below), we can avoid having to make this line change.